r/fratPhyloseq.Rmd

```diff
@@ -0,0 +1,63 @@
+---
+title: "Phyloseqify the Fratjuice samples"
+author: "Richard Sprague"
+date: "1/5/2018"
+output: github_document
+---
+
+Load the prerequisites:
+
+```{r}
+library(phyloseq)
```

Comment:
Do we need these library statements at all? I'd prefer if all namespaces were explicitly referred to (like how you did with `actino::phyloseq_from_JSON_at_rank`). Removing the library imports is often a good way to enforce specifying a function provenance.